test/sparse/CS/multiplication/correctness.chpl

```diff
@@ -126,7 +128,7 @@ proc multiply(A: [?ADom] ?eltType, B: [?BDom] eltType) where isSparseArr(A) && i
     }
   }
 
-  var indices: [1..0] 2*int;
+  var indices: list(2*int);
```

Comment:
@e-kayrakli - I believe this test could also benefit from buffered index addition, however it's not incredibly important for us to keep performance optimal for this test.